package.json

```diff
@@ -12,6 +12,7 @@
   "main": "src/index.js",
   "scripts": {
     "banners": "./utils/maintain-banners.js",
+    "prebuild": "rm -rf ./src/models ./src/factories ./src/generated-client.js",
```

Comment:
We often use a module called `rimraf` for this. I'm guessing for cross-platform compatibility. Probably nobody is going to be running build on windows, but maybe a good practice to be cross-platform compatible.

Reply:
Will change, was just following another script in repo.

Reply:
Scripts updated for `prebuild` and `predocs`